client/src/pages/Messages/Chat/Chat.tsx

```diff
@@ -0,0 +1,113 @@
+import React, { useState } from 'react';
+import { Avatar, Box, Button, FilledInput, FormControl, Typography } from '@mui/material';
+import moment from 'moment';
+import { Badge } from '@mui/material';
+import createNewMessage from '../../../helpers/APICalls/createNewMessage';
+import useStyles from './useStyles';
+import SendIcon from '@mui/icons-material/Send';
+
+const Chat = (conversation: any) => {
```

Comment:
Please also avoid using `any` as a type, if there's an interface for conversation we could use that here instead